_posts/2018-10-29-schedule-sneak-peek-rome.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "This Week in RustFest 3: Schedule sneak peek"
+socialImageSrc: "/assets/posts/rome/twirf-rustbridge.jpg"
+authors:
+- dns2utf8
+---
+
+The schedule is long awaited and it is the one thing the organisation has to make sure is interesting and digestible.
+Plus the topics should be varied.
+
+So where is the schedule? Since the talks are already [published on the website](https://rome.rustfest.eu/talks/), how hard can it be?
+Well here is the current draft:
+
+| Time  | Talk or Event      |
+|-------|--------------------|
+| 08:30 | Registration opens |
+| 09:00 | Introduction from local orga team + sponsors |
+| 09:30 | Opening keynote |
+| ...   | ... |
+| 10:45 | Coffee break |
+| ...   | ... |
+| 12:30 | Lunch break |
+| 14:00 | Afternoon talks |
+| ...   | ... |
+| 15:05 | Coffee break |
+| ...   | ... |
+| 16:55 | Closing keynote |
+| 17:40 | Tips for Rome, the Workshops and final words from the team |
+| 17:55 | Family photo |
+| 18:00 | Doors close |
+
+You might be wondering "did they just write times like `:40` and `:05`?"
+Yes we did!
+There are some Swiss people in the orga and to maximize the amount of talks we will be punctual and on time.
+
+If you are a speaker, we will give you hints on when to come to an end.
+Also, if you would like to allow the audience to ask questions, that has to fit into you slot.
+
+If you are an attendee, please be aware that talks will start definitively on time and - unless they finish ahead of the allotted time slot - there will be time only for a quick question or two.
+Speakers will be around after their talks for in depth discussions.
+
+
+# We still sell tickets
+
+Our final batch of tickets is 🎟 [live on tito](https://ti.to/asquera-event-ug/rustfest-rome-2018/) 🎟.
+
+These tickets will stay around until they sell out or shortly before the conference.
+
+# Our Friends at Threema
+
+We’re happy to welcome Threema as a sponsor for Rome.
+Threema is a Swiss🇨🇭 company building secure messaging apps.
+Their [push relay](https://github.com/threema-ch/push-relay) written in Rust is running in production and waking up countless phones daily.
+For their Android to WebApplication connection they opensourced [SaltyRTC](https://github.com/saltyrtc/saltyrtc-client-rs) an end-to-end encrypted signalling protocol such as setting up a WebRTC or ORTC peer-to-peer connection.
+As of today their iPhone app contains rust code handling the threema messaging protocol.
```

Comment:
cc @dbrgn

Reply:
Cool, thank you!

Reply:
That's cool :-) But I wonder how relevant the "a few hours ago" phrasing will be in a couple of days/weeks. Perhaps an absolute time reference, something like "And at the end of October, they released the first version ..."?

(bikeshedding)

Reply:
Blogposts, especially news posts, are always relative to the posting time, aren't they? In a few weeks, this post will simply not be relevant anymore :slightly_smiling_face:

Reply:
makes sense